### ISSUE TITLE
Question about the difference of the GRU4Rec and GRU4Rec+

### ISSUE THREAD

**Issue by zyh981022** [hungthanhpham94/GRU4REC-pytorch#9]
Hi,I have a question about how to run gru4rec and gru4rec+ with your code.From the paper,we can know these two methods vary in Loss Function and Sampling Strategy while the former is  obvious in your code,I wonder how to apply the different sampling strategy.Thanks.

**Comment by XiaoJing-C** [comment]
I have the same problem as you. Did you solve this problem?

**Comment by XiaoJing-C** [comment]
> Hi,I have a question about how to run gru4rec and gru4rec+ with your code.From the paper,we can know these two methods vary in Loss Function and Sampling Strategy while the former is obvious in your code,I wonder how to apply the different sampling strategy.Thanks.

I have the same problem as you. Did you solve this problem?
